openedx/core/lib/courses.py

```diff
@@ -27,8 +27,8 @@ def course_image_url(course, image_key='course_image'):
         else:
             url += '/images/' + image_key + '.jpg'
     elif not getattr(course, image_key):
-        # if image_key is empty, use the default image url from settings
-        url = settings.STATIC_URL + settings.DEFAULT_COURSE_ABOUT_IMAGE_URL
+        # if image_key is empty, set empty url by default
+        url = ""
```

Comment:
Is broken image an issue? In that case we can ensure a correct default image exists instead of making this change?

Reply:
I agree with @naeem91. Also, It was decided to use this image as default image as part of UI Refresh https://github.com/mckinseyacademy/mcka_apros/blob/development/static_v2/image/default-course-image.png 

Let's add this image in static files instead.

Reply:
Thanks for the suggestions. An authoring based solution was devised to the reporter using the default image provided above.